components/clm/src/biogeochem/CNCarbonFluxType.F90

```diff
@@ -3206,7 +3212,12 @@ subroutine InitHistory(this, bounds, carbon_type)
             avgflag='A', long_name='annual sum of column-level NPP', &
             ptr_col=this%annsum_npp_col, default='inactive')
 
-       
+       ! C4MIP output variable, plant carbon flux to cwd (a part of fVegLitter)
+       this%plant_c_to_cwdc(begc:endc) = spval
+       call hist_addfld1d (fname='fVegCWDC', units='gC/m^2/s', &
```

Comment:
Is there a reason why the output variable name is not in all caps? Also, should we name the output variable to be `VEG_C_TO_CWDC`?

Reply:
Thanks for the suggestion, I have renamed the fluxes to VEGC_TO_CWDC, also added VEGN_TO_CWDN, VEGP_TO_CWDP fluxes for corresponding N and P.